doc/articles/guides/xf-migration/intro.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: Uno.XamarinFormsMigration.Intro
+---
+
+# Uno Platform for Xamarin.Forms developers
+
```

Comment:
I think worth adding a section here talking about the support horizon for Xamarin.Forms and how Uno is a great option for applications to migrate forward to

Reply:
I included some of what I wrote for the overview to start the introduction page.

Despite the confusing file names, the purpose between them seems important enough, but I'd be open to [feedback](https://github.com/unoplatform/uno/pull/13984/files#diff-920e36d3600874eff58d1150a32458b87f0497246d296fb28d3fe7d0a2561704R1-R4). 

**Just to visualize what the current placement looks like:**
```yaml
- name: Overview
  href: xref:Uno.XamarinFormsMigration.Overview
- name: Uno Platform for Xamarin.Forms developers
  href: xref:Uno.XamarinFormsMigration.Intro
```